src/dbos.ts

```diff
@@ -172,7 +175,7 @@ export class DBOS {
     return await DBOSRuntime.loadClasses(dbosEntrypointFiles);
   }
 
-  static async launch(httpApps?: DBOSHttpApps) {
+  static async launch(httpApps?: DBOSHttpApps, conductorParams?: ConductorParams) {
```

Comment:
These have to be merged into one big "options" object, we're going to have more of these and some will be optional.